src/test/java/io/moderne/organizations/OrganizationDataFetcherTest.java

```diff
@@ -1,37 +1,42 @@
 package io.moderne.organizations;
 
 import com.netflix.graphql.dgs.autoconfig.DgsAutoConfiguration;
-import io.moderne.organizations.types.*;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import reactor.test.StepVerifier;
 
-import java.util.List;
+import static org.assertj.core.api.Assertions.assertThat;
 
-@SpringBootTest(classes = {DgsAutoConfiguration.class, CustomScalarsConfiguration.class, OrganizationDataFetcher.class})
+@SpringBootTest(classes = {DgsAutoConfiguration.class, CustomScalarsConfiguration.class, OrganizationDataFetcher.class, OrganizationStructureService.class})
 public class OrganizationDataFetcherTest {
 
     @Autowired
     OrganizationDataFetcher organizationDataFetcher;
 
     @Test
     void organizationForThisRepository() {
-        Organization all = Organization.newBuilder()
-                .id("ALL")
-                .name("ALL")
-                .commitOptions(List.of(CommitOption.Direct, CommitOption.Branch, CommitOption.Fork, CommitOption.PullRequest, CommitOption.ForkAndPullRequest))
-                .build();
         StepVerifier
-                .create(organizationDataFetcher.organizations(new RepositoryInput("openrewrite/rewrite", "github.com", "main")))
-                .expectNext(
-                        Organization.newBuilder()
-                                .id("OpenRewrite")
-                                .name("OpenRewrite")
-                                .commitOptions(List.of(CommitOption.PullRequest, CommitOption.Branch, CommitOption.ForkAndPullRequest, CommitOption.Fork))
-                                ._parent(all)
-                                .build())
-                .expectNext(all)
+                .create(organizationDataFetcher.allOrganizations())
```

Comment:
ALL should continue to be atop the hierarchy.

Reply:
Do you mean it has to come first in the graphql response? Or just still be the top level?

It currently is the top level organization, but it's coming from the repos.csv now.

Reply:
There was a comment in the code saying that `default` is the first to be returned so that UI selects that by default.
Before this change ALL was not returned first it seems.

Reply:
I've put ALL on top now, and add all repos explicitly to it.

Reply:
> ALL should continue to be atop the hierarchy.

This should be up to the repos.csv to determine, especially since we also have an "orphaned" now.

Reply:
then the question remains: How?
Do we take the first orgN and make that the first then orgN-1 until org1, then the next?
Currently it's the other way around (org1 -> orgN) but that would be simple enough to change.